{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/account/apps.py

```diff
@@ -0,0 +1,5 @@
+from django.apps import AppConfig
+
+class AccountConfig(AppConfig):
+	name = "account"
```

Comment:
Similarly I think we have to name everything `{{ cookiecutter.repo_name }}.$app_name`